semantic_model_generator/validate/context_length.py

```diff
@@ -0,0 +1,15 @@
+_MODEL_CONTEXT_LENGTH = 7000  # We use 7k so that we can reserve 1k for response tokens.
```

Comment:
We should reduce this to also account for the tokens consumed by our prompts

Reply:
great call - updated